setup.py

```diff
@@ -319,7 +319,7 @@ def write_version(filename: str = os.path.join(*[my_dir, "airflow", "git_version
     'pandas-gbq<0.15.0',
 ]
 grpc = [
-    'google-auth>=1.0.0, <2.0.0dev',
+    'google-auth>=1.0.0, <3.0.0dev',
```

Comment:
(Not directly related to this PR) @potiuk Why was the `dev` suffix added in the first place and can we remove it? This seems to be the only place we are specifying prereleases explicitly (in cdb3f25456e49d0199cd7ccd680626dac01c9be6).

Reply:
Good point. I have no idea :). I added it > year ago, and without suitable comment (my bad). But I am quite sure, whatever it was then it is not applicable any more and we can change it to <3.0.0 - providing that it passes all the tests.

Reply:
@busunkim96 Could you try removing `dev` from the version constraints?

Reply:
Done.